src/components/Layout/Layout.sass

```diff
@@ -0,0 +1,51 @@
+@import '../../stylesheets/config'
+
+.main
+  +flex-container(column)
+  background-color: $color-neutral-800
+  min-height: 100vh
+
+.header
+  +flex-container(column, flex-start, center)
+  +container
+  color: $color-white
+  padding-top: $m-sm
+
+  &__nav
+    +flex-container(row, flex-start, center)
+    gap: $m-sm
+
+  &__nav-links
+    +flex-container(row)
+    gap: $m-sm
+
+  &__nav-link
+    color: $color-white
+    padding-bottom: $m-xs
+
+    &:hover
+      border-bottom: 1px solid $color-border
+      color: $color-white
+  
+  &__nav-search
+    background-color: $color-neutral-800
+    border: 2px solid darken($color-border, 20)
```

Comment:
Try to avoid using darken and lighten properties, it generates additional colors, the entire color scheme should include an action such as hover etc.